Rb.Forms.Barcode.Pcl/BarcodeScanner.cs

```diff
@@ -246,6 +246,10 @@ public void OnCameraReleased()
 
         private static void OnBarcodeChanged(BindableObject bindable, Barcode oldValue, Barcode newBarcode)
         {
+            if (newBarcode == null) {
+                return;
+            }
```

Comment:
Hmmm actually that check shouldnt be necessary as the Barcode object has an equal check: https://github.com/rebuy-de/rb-forms-barcode/blob/master/Rb.Forms.Barcode.Pcl/Barcode.cs#L33-L50